readme.md

```diff
@@ -144,6 +147,12 @@ Accepts keywords which ends up as a path in Analytics.
 
 `.track('init', 'backbone')` becomes `/init/backbone`
 
+#### .trackEvent(category, action, [label, value])
```

Comment:
I also lean towards following GA signature for this function.

I think it might seems messy seen like this without context, but there's an important characteristic that actually make this signature the best one for this function. The main point is that all these arguments are actually graduals. This mean that you cannot specify a label without an action, or a value without a label, etc, etc. Each previous arguments is required for the following one.

GA uses these values to filter deeper inside an event. You start by looking at the category then the action, and if it makes sense you the dig even further with a label and then a value.